wrangler.toml

```diff
@@ -0,0 +1,7 @@
+name = "helloworld"
+type = "javascript"
```

Comment:
Should we make default webpack instead?

Reply:
Probably. It'd also be great to move `index.js` from the top level and create a `src` directory and point `package.json`:`main` to `./src/index.js`

Reply:
Ill make an issue for that

Reply:
https://github.com/cloudflare/worker-template/issues/26